backend/pkg/schema/client.go

```diff
@@ -697,9 +697,18 @@ func (c *Client) GetSchemasIndividually(ctx context.Context, showSoftDeleted boo
 	}
 	ch := make(chan chRes, len(subjectsRes.Subjects))
 
+	// limit max concurrency to 10 requests at a time
+	guard := make(chan struct{}, 10)
+
 	// Describe all subjects concurrently one by one
 	for _, subject := range subjectsRes.Subjects {
+		guard <- struct{}{}
+
 		go func(s string) {
+			defer func() {
+				<-guard
+			}()
```

Comment:
Can we refactor to use errgroup with SetLimit here?

Reply:
Updated.